packages/kbn-i18n/BUILD.bazel

```diff
@@ -102,38 +102,16 @@ ts_project(
   allow_js = True,
   declaration = False,
```

Comment:
why `declaration: false` for the server-side code?

Reply:
That `declaration: false` is within respect to the browser code build not to the node one :)